src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,9 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    public int compareTo(Person otherPerson) {
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.